frontend/src/app/components/search/search.component.html

```diff
@@ -47,7 +47,7 @@
     <div class="search-grid flex-row">
       <md-icon>search</md-icon>
       <md-input-container floatPlaceholder="never" class="search-box">
-        <input #searchBox mdInput placeholder="Name, Username or Roll" (keyup)="searchTerm(searchBox.value)" [disabled]="loading">
+        <input #searchBox mdInput placeholder="Name, Username or Roll" type="text" (keyup)="searchTerm(searchBox.value)">
```

Comment:
I think, not letting type is actually worse ux, we could always handle searching when the content fully loads. Also this was necessary because I was not able call focus when disable was added (I suppose it's bug with mdInput directive.